app/ide-desktop/lib/dashboard/src/layouts/AssetPanel.tsx

```diff
@@ -134,6 +141,25 @@ export default function AssetPanel(props: AssetPanelProps) {
               {getText('versions')}
             </ariaComponents.Button>
           )}
+        {item != null && item.item.type === backendModule.AssetType.project && (
+          <ariaComponents.Button
+            size="medium"
+            variant="ghost"
+            className={tailwindMerge.twMerge(
+              'button pointer-events-auto select-none bg-frame px-button-x leading-cozy transition-colors hover:bg-selected-frame',
```

Comment:
oh what's that? why do you provide that much of custom classes together with `variant` prop?